beets/ui/commands.py

```diff
@@ -823,6 +823,8 @@ def _get_choices(self, task):
         Returns a list of `PromptChoice`s.
         """
         # Standard, built-in choices.
+        #Bell ring to alert the user that he has to interfere with the cmd
```

Comment:
This Travis report: https://travis-ci.org/beetbox/beets/jobs/215295136#L885

is telling you to put a space after the `#` character in your comments. In addition, please use complete sentences in comments that end with periods—and there's no need to use abbreviations like "cmd".